disk/disk.go

```diff
@@ -154,6 +154,15 @@ func (d *Disk) Remove(name string, all bool) error {
 	return os.RemoveAll(name)
 }
 
+func (d *Disk) Sync(name string) error {
```

Comment:
Good question. 

1. In the near future, write will write into a cache (OS page cache or user level cache) by default.

2. If you look at the outer API, we want to be able to sync multiple files at once. So we cannot simply add a field into write.